spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'associations' do
+    it 'belongs to an author' do
+      association = described_class.reflect_on_association(:author)
+      expect(association.macro).to eq(:belongs_to)
+      expect(association.options[:class_name]).to eq('User')
+    end
+
+    it 'belongs to a post' do
+      association = described_class.reflect_on_association(:post)
+      expect(association.macro).to eq(:belongs_to)
+    end
+  end
+end
```

Comment:
Nice job so far. However, as part of the project requirements, any custom method in the models should have a corresponding unit test. Kindly consider to add a unit test for the `update_comment_count` method in the comment model here. :pray: